apps/wizard/app_pages/chart_diff/conflict_resolver.py

```diff
@@ -153,7 +152,7 @@ def resolve_conflicts(self, rerun: bool = False):
                 config.pop(field, None)
 
             # Verify config
-            config_new = validate_chart_config_and_set_defaults(config, schema=get_schema("004"))
+            config_new = validate_chart_config_and_set_defaults(config, schema=get_schema_from_url(config["$schema"]))
```

Comment:
could it be the case that this raised a `KeyError` exception bc `$schema` is not a property in `config`?

Reply:
Good question. Fortunately, `$schema` is never null.